joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -102,28 +109,46 @@ controller_interface::return_type
 JointTrajectoryController::update()
 {
   if (get_current_state().id() == State::PRIMARY_STATE_INACTIVE) {
-    if (!is_halted) {
+    if (!is_halted_) {
       halt();
-      is_halted = true;
+      is_halted_ = true;
     }
     return controller_interface::return_type::SUCCESS;
   }
 
   auto resize_joint_trajectory_point =
-    [](trajectory_msgs::msg::JointTrajectoryPoint & point, size_t size)
+    [&](trajectory_msgs::msg::JointTrajectoryPoint & point, size_t size)
     {
       point.positions.resize(size);
       point.velocities.resize(size);
       point.accelerations.resize(size);
```

Comment:
I think these two lines are redundant with the new lines you added (L128 & L133)

Reply:
Thanks! Very good catch!